firebase-app-distribution/src/test/java/com/google/firebase/appdistribution/AabUpdaterTest.java

```diff
@@ -84,7 +98,8 @@ public void setup() throws IOException, FirebaseAppDistributionException {
 
     aabUpdater =
         Mockito.spy(
-            new AabUpdater(mockLifecycleNotifier, mockHttpsUrlConnectionFactory, testExecutor));
+            new AabUpdater(
+                firebaseApp, mockLifecycleNotifier, mockHttpsUrlConnectionFactory, testExecutor));
```

Comment:
If you take my suggestion and require a `Context` here instead of a `FirebaseApp`, then you can just pass in `ApplicationProvider.getApplicationContext()`.